commit 56676709a8a0ecf51429bc1ac508513f72d0a761
Author: Andreas Sturmlechner <andreas.sturmlechner@gmail.com>
Date:   Sun Mar 13 14:43:50 2016 +0100

    Make Qt5Test really optional, fix plasmate dependencies
    
    KF5NewStuff, KF5Parts are only required for plasmate.
    Could not push KF5ItemModels down into plasmate without breaking
    build. Is it a problem within KDevPlatform?

diff --git a/CMakeLists.txt b/CMakeLists.txt
index b556de6..bc982ae 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -74,7 +74,7 @@ endif()
 # Load CMake, Compiler and InstallDirs settings from KF5 and the following are already somewhat "done" tier1/tier2 libs from kdelibs:
 find_package(KF5 REQUIRED COMPONENTS
     Archive Completion Config ConfigWidgets CoreAddons DBusAddons Declarative I18n IconThemes ItemModels
-    KIO NewStuff Parts Plasma PlasmaQuick Service TextEditor WidgetsAddons XmlGui WindowSystem)
+    KIO Plasma PlasmaQuick Service TextEditor WidgetsAddons XmlGui WindowSystem)
 
 #########################################################################
 
diff --git a/cuttlefish/CMakeLists.txt b/cuttlefish/CMakeLists.txt
index 6eaa78a..5b90fef 100644
--- a/cuttlefish/CMakeLists.txt
+++ b/cuttlefish/CMakeLists.txt
@@ -2,7 +2,9 @@ project(cuttlefish)
 add_definitions(-DTRANSLATION_DOMAIN=\"cuttlefish\")
 
 add_subdirectory(src)
-add_subdirectory(autotests)
+if(BUILD_TESTING)
+    add_subdirectory(autotests)
+endif()
 
 
 plasma_install_package(package org.kde.plasma.cuttlefish packages package)
diff --git a/plasmate/CMakeLists.txt b/plasmate/CMakeLists.txt
index 8f28ec4..4586102 100644
--- a/plasmate/CMakeLists.txt
+++ b/plasmate/CMakeLists.txt
@@ -13,6 +13,8 @@ add_feature_info(GIT_EXECUTABLE_FOUND "Git" "Distributed Revision Control System
 
 find_package(KF5 REQUIRED COMPONENTS
     KDELibs4Support
+    NewStuff
+    Parts
 )
 
 #install( FILES previewer/windowswitcher/thumbnails/konqueror.png
